db_scripts/create_tables.sql

```diff
@@ -90,11 +110,27 @@ CREATE TABLE socceroffsides (
 	GameId uuid,
 	TeamFor character varying(255) NOT NULL,
 	TeamAgainst character varying(255) NOT NULL,
-    offsidetime integer NOT NULL
+    offsidetime integer NOT NULL,
+
+    CONSTRAINT fk_soccer_offsides_to_game FOREIGN KEY(GameId) REFERENCES soccergames(id)
 );
 
 CREATE INDEX IF NOT EXISTS soccer_off_sides_pkey ON socceroffsides(id uuid_ops);
 
+------------------------------------------------------------------
+
+CREATE TABLE soccergameends (
```

Comment:
@KnotScientific @babrar when do we want to populate this table?

Reply:
End of game, if that's what you mean? I send out a request with the final score (there will be ammendments, like decreasing the scores if someone accidently increases it, etc).

Reply:
So create a new endpoint?

Reply:
The endpoint stays the same, just have to add an aditional if statement on how to handle like event_type == 'end' or something.
<img width="305" alt="image" src="https://user-images.githubusercontent.com/29667155/221060299-d76dca8f-64e1-46a5-8114-ecb4a163e103.png">

Reply:
Gotcha.

Reply:
Added in starter code for this if-block. Since it's not a breaking change, a new event_type will be handled in a new PR. @madhur4444 will create a ticket and assign it to you.